core/src/main/scala/org/http4s/MediaType.scala

```diff
@@ -176,320 +187,176 @@ sealed class MediaType(
   override def satisfiedBy(mediaType: MediaRange): Boolean = mediaType match {
     case mediaType: MediaType =>
       (this eq mediaType) ||
-        mainType == mediaType.mainType &&
-          subType == mediaType.subType
+        mainType === mediaType.mainType &&
+          subType === mediaType.subType
 
     case _ => false
   }
 
   override def equals(obj: Any): Boolean = obj match {
     case x: MediaType =>
       (this eq x) ||
-        mainType == x.mainType &&
-          subType == x.subType &&
-          extensions == x.extensions
+        mainType === x.mainType &&
+          subType === x.subType &&
+          extensions === x.extensions
     case _ => false
   }
 
-  override def hashCode(): Int = renderString.##
-  override def toString: String = "MediaType(" + renderString + ')'
-}
-
-object MediaType extends Registry {
```

Comment:
Bye, bye `Registry`. But also `registerFileExtension` is gone

Reply:
It won't be missed.